lgc/interface/lgc/Pipeline.h

```diff
@@ -115,6 +115,7 @@ struct Options {
   unsigned nggPrimsPerSubgroup;        // How to determine NGG prims per subgroup
   unsigned shadowDescriptorTable;      // High dword of shadow descriptor table address, or
                                        //   ShadowDescriptorTableDisable to disable shadow descriptor tables
+  unsigned robustAccess; // Give defined behavior for null buffer descriptor and out-of-range buffer load and store
```

Comment:
The indent of the comment will probably fail the clang-format check.

Reply:
It is weired that git clang-format gives the above result. I manually do the indent.